unicode_smilies/unicode_smilies.php

```diff
@@ -14,586 +14,583 @@ function unicode_smilies_install() {
 }
 
 function unicode_smilies_smilies(&$a,&$b) {
-	Smilies::add($b, ':-)', '&#x1F600;');
-	Smilies::add($b, ':)', '&#x1F600;');
-	Smilies::add($b, ':-(', '&#x1F641;');
-	Smilies::add($b, ':(', '&#x1F641;');
-	Smilies::add($b, ':-[', '&#x1F633;');
-	Smilies::add($b, ':-D', '&#x1F601;');
-	Smilies::add($b, ':D', '&#x1F601;');
-	Smilies::add($b, ';-)', '&#x1F609;');
-	// Smilies::add($b, ';)', '&#x1F609;'); // Deactivated since this leads to disturbed html entities
-	Smilies::add($b, ':-P', '&#x1F61B;');
-	Smilies::add($b, ':-p', '&#x1F61B;');
-	Smilies::add($b, ':P', '&#x1F61B;');
-	Smilies::add($b, ':p', '&#x1F61B;');
-	Smilies::add($b, ':-O', '&#x1F62E;');
-	Smilies::add($b, ':O', '&#x1F62E;');
-	Smilies::add($b, ':-X', '&#x1F910;');
-	Smilies::add($b, ':-x', '&#x1F910;');
-	Smilies::add($b, ':X', '&#x1F910;');
-	Smilies::add($b, ':x', '&#x1F910;');
-	Smilies::add($b, '8-)', '&#x1F60E;');
-	Smilies::add($b, ':-/', '&#x1F615;');
-	Smilies::add($b, ':-"', '&#x1F48F;');
+	Smilies::add($b, ':-):', '&#x1F600;');
```

Comment:
While it is correct that the emoticon names are enclosed by a pair of `:` at the beginning and the end, I don't think the same should be done with plain ASCII emoticons.

I think the problem with the `:p` you mentioned is rather, that two letter representations of the emoticons should not be replaced at all as their combination is to common in the normally used language.

Reply:
The plain ASCII emoticons are not enclosed by a :